rubato/game.py

```diff
@@ -155,6 +155,37 @@ def update(cls):  # test: skip
                     },
                 )
 
+            if event.type in (sdl2.SDL_MOUSEBUTTONDOWN, sdl2.SDL_MOUSEBUTTONUP):
+                mouse_button = None
+                if event.button.state == sdl2.SDL_BUTTON_LEFT:
+                    mouse_button = "mouse 1"
```

Comment:
This naming doesn't match what you have in the docs

Reply:
Lol fixed in later commits in the original branch.